cmd/cosign/webhook/main.go

```diff
@@ -85,6 +87,9 @@ var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 }
 
 func NewValidatingAdmissionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+	// Decorate contexts with the current state of the config.
```

Comment:
@vaikas Shouldn't you do the same for the NewMutatingAdmisisonController ?

Reply:
lol, I took it out as per previous PR :) Is there anything in the policy that gets used by mutating?